browsing-context.md

```diff
@@ -1,28 +1,290 @@
 # Prerendering browsing contexts
 
-TODO placeholder. Grab stuff from portals.
+We envision modernized prerendering to work by loading content into a **prerendering browsing context**. A prerendering browsing context can be thought of as a tab that is not yet shown to the user, and which the user has not yet affirmatively indicated an intention to visit. As such, it has additional restrictions placed on it to ensure the user's privacy and prevent disruptions.
```

Comment:
nit: Implied by the next paragraph, but maybe explicitly refer to a prerendering browsing context as a top level browsing context.